frontend/src/pages/recommend/Recommend.tsx

```diff
@@ -1,25 +1,66 @@
 import React from 'react';
-
+import Image from 'next/image';
 import { db } from '../../lib/firebase';
 
 const uuId = 'User4KpZPzCR6zJy0KUX';
 let res_recommend = '';
 
 let Recommend_channel: Array = null;
 let Recommend_channel_len: Array = null;
+let Recommend_user: Array = null;
+let Recommend_user_atribute: Array = null;
+let Recommend_user_len: Array = null;
+
+function handleOnClick_join_channel(e) {
+  //POST request URL
+  //'https://httpbin.org/post'
+  const POST_request_URL = 'https://httpbin.org/post';
```

Comment:
実装が遅くなって申し訳ないのですが，チャンネル参加 API のエンドポイントを `http://localhost:8000/api/joinChannel` に作ったので，リクエスト先をそこに変更してもらえると良いかもです．#16 

```suggestion
  const POST_request_URL = 'http://localhost:8000/api/joinChannel';
```

Reply:
実装ありがとうございます！
リクエスト先を変更しました